templates/mon_schema.sql.j2

```diff
@@ -0,0 +1,29 @@
+-- Tune the DB
+SELECT SET_CONFIG_PARAMETER ('MaxClientSessions', 200);
+-- Turn off messages in the log created by the load balancer/icinga checks
+SELECT set_config_parameter('WarnOnIncompleteStartupPacket', 0);
+
+-- Enable SSL ** Requires a db restart, also the restart will fail of the ssl cert is not in place on the server
+-- The certs are placed in the root catalog dir by vertica and should be linked to the correct dir after db creation
+-- ln /var/vertica/catalog/server* /var/vertica/catalog/mon/v*/'
+SELECT SET_CONFIG_PARAMETER('EnableSSL', '1');
```

Comment:
This requires the ssl certs to be in place and requires a restart after it is done. Given both of those I think it should be done during db creation with the enabling of ssl.